src/Link/Link.tsx

```diff
@@ -22,9 +22,9 @@ const hoverColor = system({
 
 const StyledLink = styled.a<StyledLinkProps>`
   color: ${props => (props.muted ? get('colors.fg.muted')(props) : get('colors.accent.fg')(props))};
-  text-decoration: ${props => (props.underline ? 'underline' : 'none')};
+  text-decoration: ${props => (props.underline ? 'underline' : 'var(--prefers-link-underlines, underline)')};
```

Comment:
Assuming `BaseStyles` is an optional wrapper... (correct me if it's not!)

Instead of changing the default, should we keep that default and let folks change the default by _switching-on_ `--prefers-link-underlines`? 

```ts
text-decoration: props.underline ? 'underline' : 'var(--prefers-link-underlines, none)'
```

If `--prefers-link-underlines` is not defined, for example when BaseStyles is not imported, the fallback would be none

Reply:
@joshblack @pksjce Does this sound right or am I missing the goal here?